modules/reference/pages/rpk/rpk-transform/rpk-transform-deploy.adoc

```diff
@@ -0,0 +1,51 @@
+= rpk transform deploy
+:description: rpk transform deploy
+
+Deploy a transform.

+

+When run in the same directory as a transform.yaml it will read the configuration file,

+then look for a .wasm file with the same name as your project. If the input and output

+topics are specified in the configuration file, those will be used, otherwise the topics

+can be specified on the command line using the --input-topic and --output-topic flags.

+

+Wasm files can also be deployed directly without a transform.yaml file by specifying it

+like so:

+

+  rpk transform deploy transform.wasm --name myTransform

+

+Environment variables can be specified for the transform using the --env-var flag, these

+are separated by an equals for example: --env-var=KEY=VALUE


```

Comment:
@micheleRP based on the example the '=' separates the key and the value not the variables from each other.  Also we don't show an example with multiple env variables which are actually separated (by what I dont know).

cc @rockwotj